frontend/src/component/feature/FeatureView/FeatureSettings/FeatureSettingsProject/FeatureProjectSelect/FeatureProjectSelect.tsx

```diff
@@ -30,6 +30,9 @@ const FeatureProjectSelect = ({
             key: project.id,
             label: project.name,
             title: project.description,
+            sx: {
+                textWrap: 'auto',
```

Comment:
Even though the popover is `width: min-content`, it won't wrap items in the dropdown unless you specifically ask it to. As such, we need to tell it to wrap project names.

This is pretty much only going to be relevant with project names (I think they're the only resource where we don't constrain lengths at all (even though we probably should)), so I don't expect the change in the `GeneralSelect` component to be noticeable at all in most cases.